libraries/Microsoft.Bot.Schema/ActivityEx.cs

```diff
@@ -588,9 +589,7 @@ protected bool IsActivity(string activityType)
                 if (!result)
                 {
                     // Finally, if the type is longer than the type they're looking for then we need to check if there's a / separator right after the type they're looking for
-                    result = type.Length > activityType.Length
```

Comment:
* Deleted condition `type.Length > activityType.Length`, because in order for it to reach this line in the first place, it needs to have:
		1. started with the `activityType` (e.g. starts with `"message"`)
		2. Also not be the same length as `activityType`

* There is no way for it to possibly start with `activityType` and be *shorter* (if it's shorter, it would not have satisfied the "startsWith" condition, therefore will have never reached the comparison of lengths)